bindings/matlab/iio-wrapper.h

```diff
@@ -1,4 +1,56 @@
 #ifndef MATLAB_LOADLIBRARY
 #define MATLAB_LOADLIBRARY
 #include <iio.h>
+
+#ifdef _WIN32
+#   ifdef LIBIIO_EXPORTS
+#	define __api __declspec(dllexport)
+#   else
+#	define __api __declspec(dllimport)
+#   endif
+#elif __GNUC__ >= 4 && !defined(MATLAB_MEX_FILE) && !defined(MATLAB_LOADLIBRARY)
+#   define __api __attribute__((visibility ("default")))
```

Comment:
I didn't modify that part-- I believe it was added in b20f340421.

It's an odd format (maybe __MATLAB_IIO_WRAPPER__ would be better?), but I believe the ifdefs exist to prevent multiple inclusion. This header should only be included if the MATLAB bindings are being (1) built into the library or (2) called from MATLAB